pkg/utils/config_checker.go

```diff
@@ -0,0 +1,105 @@
+package utils
```

Comment:
Could you please add the copyright header and also add a comment explaining where this was copied from and why it was copied rather than imported?

Reply:
done